delivery_purchase_label/data/stock_picking_type.xml

```diff
@@ -0,0 +1,9 @@
+<odoo noupdate="1">
+    <record id="picking_type_send_label" model="stock.picking.type">
+        <field name="name">Dropship Carrier Label</field>
+        <field name="code">internal</field>
+        <field name="sequence_code">LBL</field>
+        <field name="default_location_dest_id" ref="stock.stock_location_suppliers" />
```

Comment:
Why is there the suppliers location used?
And not customers?
And why is the code internal and not outgoing?
What are the reasons for that? Stock valuation?
```suggestion
        <field name="default_location_dest_id" ref="stock.stock_location_suppliers" />
```

Reply:
Yes and it is good for valuation layer, because if it would be supplier -> customer is would be recognized as dropship and will create a valuation for it.